chapters/data-analysis.tex

```diff
@@ -1,86 +1,96 @@
 %------------------------------------------------
 
 \begin{fullwidth}
-Transforming raw data into a substantial contribution to scientific knowledge 
-requires a mix of subject expertise, programming skills, 
-and statistical and econometric knowledge. 
-The process of data analysis is, therefore, 
-a back-and-forth discussion between people 
-with differing skill sets. 
-The research assistant usually ends up being the pivot of this discussion. 
+Transforming raw data into a substantial contribution to scientific knowledge
+requires a mix of subject expertise, programming skills,
+and statistical and econometric knowledge.
+The process of data analysis is, therefore,
+a back-and-forth discussion between people
+with differing skill sets.
+The research assistant usually ends up being the pivot of this discussion.
 It is their job to translate the data received from the field into
-economically meaningful indicators and to analyze them 
+economically meaningful indicators and to analyze them
 while making sure that code and outputs do not become too difficult to follow or get lost over time.
+This can be a complex process.
 
-When it comes to code, though, analysis is the easy part, 
-\textit{as long as you have organized your data well}. 
-Of course, there is plenty of complexity behind it: 
+When it comes to code, though, analysis is substantially less intricate than data cleaning,
+as long as you have already organized your data well.
```

Comment:
I am not sure if this is the best way to put this. But since this will conflict with #375 I think there will just create more mess to try to edit this here.